### DISCUSSION TITLE
Recorded and Imported tracks

### DISCUSSION THREAD

**Discussion by Cadizfox** [osmandapp/OsmAnd#12156] [Q&A]
Hi, I have managed to import previously created and record tracks onto OSMApp and it works fine.
Can someone tell me how to delete tracks asnd msrkers once they are in OSMApp?
If this questionn has already been asked, please direct me to the discussion.
Many thanks for your help

**Comment by Ualik123** [comment]
For android (and ios maybe, i dunno, i think its same):
On main screen tap button with three lines in bottom left corner, select "my places", "tracks", you will see all your tracks.
You can delete a single track by tapping 3 dots to the right from its name and select "delete".
If you want to delete many tracks at once you can tap trash bin icon on bottom of the screen and selest tracks that you want to delete, after you select them tap trash bin in top right corner of the screen.

Or on android you can just open Android\data\net.osmand.plus\files\tracks and delete files by hand.

**Reply by Cadizfox** [reply]
Thank you very much Ualik123. It's easy if you know how. It may be an idea to add some more advanced editing features to the general documentation.
Cherrs
